### ISSUE TITLE
Please add a Compare mode?

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#191]
I very often compare 2 files, for example a list for a build, and a list I work on with arrivals or that I edit because some items are to be changed... whatever... the idea is to compare 2 lists.

Right now I use the "-" (Subtract Items) command.
The problem is that it EDITS the file you're working on.

That means that if you wish to add a comment, you can't, because you also wish to remove this substraction (with CTRL+Z).
So if you typed a comment, CTRL+Z will remove it first.

![image](https://user-images.githubusercontent.com/32078594/107858074-74af5580-6e32-11eb-8e74-1db65adb9779.png)

In short, something like Subtract but which is NOT an EDITION, so you can remove the comparaison after having commented (or remarks, or removed items, whatever...) the differences you've found?
